docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos.md

```diff
@@ -721,41 +738,51 @@ Use the following instructions on your Linux machine to customize the arguments
   |OS|Public Repo|BYOS Edge OS|`1.0.0`|
 
 
-21. Replace the the cluster profile's BYOOS pack manifest with the following custom manifest so that the cluster profile can pull the provider image from the ttl.sh image registry.  
+21. Replace the the cluster profile's BYOOS pack manifest with the output provided to your earlier that you copied.  
```

Comment:
Line 724 reads awkwardly or something is missing. "your" should probably be "you". But the larger question is whether the user is provided something or has copied something. This needs clarifying.